lib/features/children/generosity_challenge/assignments/family_values/widgets/organisation_detail_bottomsheet.dart

```diff
@@ -0,0 +1,137 @@
+                  margin:
+                      const EdgeInsets.symmetric(vertical: 12, horizontal: 24),
+                  child: Image.network(
+                    value.orgImagePath,
+                    fit: BoxFit.cover,
+                  ),
+                ),
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(20, 0, 20, 100),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      Text(
+                        value.organisation.organisationName ?? '',
+                        textAlign: TextAlign.start,
+                        style: Theme.of(context).textTheme.titleLarge?.copyWith(
+                              color: AppTheme.primary20,
+                              fontWeight: FontWeight.w700,
+                              fontFamily: 'Rouna',
+                            ),
+                      ),
+                      const SizedBox(height: 12),
+                      // should be short description
+                      Text(
+                        value.longDescription,
+                        textAlign: TextAlign.start,
+                        style: Theme.of(context).textTheme.labelLarge?.copyWith(
+                              color: AppTheme.primary20,
+                              fontWeight: FontWeight.w500,
+                              fontFamily: 'Rouna',
+                            ),
+                      ),
+                      const SizedBox(height: 16),
+                      Text(
+                        'About us',
+                        textAlign: TextAlign.start,
+                        style: Theme.of(context).textTheme.labelLarge?.copyWith(
+                              color: AppTheme.primary20,
+                              fontWeight: FontWeight.w700,
+                              fontFamily: 'Rouna',
+                            ),
+                      ),
+                      Text(
+                        value.longDescription,
+                        textAlign: TextAlign.start,
+                        style: Theme.of(context).textTheme.labelLarge?.copyWith(
+                              color: AppTheme.primary20,
+                              fontWeight: FontWeight.w500,
+                              fontFamily: 'Rouna',
+                            ),
+                      ),
+                      const SizedBox(height: 12),
+                    ],
+                  ),
+                ),
+              ],
+            ),
+          ),
+          floatingActionButtonLocation:
+              FloatingActionButtonLocation.centerDocked,
+          floatingActionButton: Padding(
+            padding: const EdgeInsets.only(bottom: 12),
+            child: GivtElevatedButton(
+              text: "Donate",
+              // isDisabled: !isDonateButtonActive,
+              onTap: () {
+                // AnalyticsHelper.logEvent(
```

Comment:
I would delete these comment. They will be in the git history if we need them later on.